diff --git a/tmux.conf b/tmux.conf
index 66b4385..916852c 100644
--- a/tmux.conf
+++ b/tmux.conf
@@ -82,20 +82,20 @@ set-option -g message-fg black
 ##+ status bar:
 set-option -g status-utf8 on
 set-option -g status-justify right
-set-option -g status-bg '#0e0e0e'
-set-option -g status-fg '#333333'
+set-option -g status-bg '#fafafa'
+set-option -g status-fg '#a3a3a3'
 set-option -g status-interval 5
 set-option -g status-left-length 60
-set-option -g status-left '#[fg=#333333]## #[fg=#008080]#T#[default]'
-set-option -g status-right '#[fg=#008080]#S:#P#[default]'
+set-option -g status-left '#[fg=#a3a3a3]## #[fg=#656565]#T#[default]'
+set-option -g status-right '#[fg=#656565]#S:#P#[default]'
 set-option -g visual-activity on
 set-window-option -g monitor-activity on
-set-window-option -g window-status-current-bg '#0e0e0e'
-set-window-option -g window-status-current-fg '#008080'
-set-window-option -g window-status-bg '#0e0e0e'
-set-window-option -g window-status-fg '#333333'
+set-window-option -g window-status-current-bg '#fafafa'
+set-window-option -g window-status-current-fg '#656565'
+set-window-option -g window-status-bg '#fafafa'
+set-window-option -g window-status-fg '#a3a3a3'
 
 ##+ pane pane border
-set-option -g pane-active-border-fg '#008080'
+set-option -g pane-active-border-fg '#656565'
 
 
